testinfra/test.py

```diff
@@ -9,11 +9,7 @@
 import sys
 import tempfile
 
-# By default let's assume we're testing against the development VM.
-try:
-    target_host = sys.argv[1]
-except IndexError:
-    target_host = "development"
+target_host = sys.argv[1]
```

Comment:
I would prefer to see us using `argparse` instead of this to make the scripts a little more documented to that we can always safely do `./script -h` and get something useful out.

Reply:
This would be a valuable addition to test.py but I would prefer it is implemented in a separate pull request.

Reply:
Also fair.